Error when following along angular.io tutorial

When following the tutorial https://angular.io/docs/ts/latest/tutorial/toh-pt4.html about services, /app.component.spec.ts breaks when we change the constructor for AppComponent.
A little bit confusing for people learning it for the first time.

```
> npm start

app/app.component.spec.ts(29,12): error TS2346: Supplied parameters do not match any signature of call target.
```

--- a/app/app.component.spec.ts
+++ b/app/app.component.spec.ts
@@ -24,12 +24,6 @@ describe('Smoke test', () => {
   });
 });
 
-describe('AppComponent with new', function () {
-  it('should instantiate component', () => {
-    expect(new AppComponent()).toBeDefined('Whoopie!');
-  });
-});
-
 describe('AppComponent with TCB', function () {
 
   it('should instantiate component',

